pybamm/parameters/parameter_sets.py

```diff
@@ -6,59 +6,76 @@
+        Peyman Mohtat, Suhak Lee, Valentin Sulzer, Jason B. Siegel, and Anna G.
+        Stefanopoulou. Differential Expansion and Voltage Model for Li-ion Batteries at
+        Practical Charging Rates. Journal of The Electrochemical Society,
+        167(11):110561, 2020. doi:10.1149/1945-7111/aba5d1.
     * NCA_Kim2011 :
-        G. H. Kim, K. Smith, K. J. Lee, S. Santhanagopalan, and A. Pesaran.
-        "Multi-domain modeling of lithium-ion batteries encompassing multi-physics in
-        varied length scales." Journal of The Electrochemical Society, 158(8), A955-A969
-        (2011).
-    * Ramadass2004 :
-        P. Ramadass, B. Haran, P. M. Gomadam, R. White, and B. N. Popov. "Development
-        of First Principles Capacity Fade Model for Li-Ion Cells." Journal of the
-        Electrochemical Society, 151(2), A196-A203 (2004).
-    * Ai 2020 :
-        > Ai, W., Kraft, L., Sturm, J., Jossen, A., & Wu, B. (2020).
-        Electrochemical Thermal-Mechanical Modelling of Stress Inhomogeneity
-        in Lithium-Ion Pouch Cells. Journal of The Electrochemical Society,
-        167(1), 013512. DOI: 10.1149/2.0122001JES.
+        Gi-Heon Kim, Kandler Smith, Kyu-Jin Lee, Shriram Santhanagopalan, and Ahmad
+        Pesaran. Multi-domain modeling of lithium-ion batteries encompassing
+        multi-physics in varied length scales. Journal of the Electrochemical Society,
+        158(8):A955–A969, 2011. doi:10.1149/1.3597614.
     * Prada2013 :
-        C.-H. Chen, F. Brosa Planella, K. O’Regan, D. Gastol, W. D. Widanage, and E.
-        Kendrick. "Development of Experimental Techniques for Parameterization of
-        Multi-scale Lithium-ion Battery Models." Journal of the Electrochemical Society,
-        167(8), 080534 (2020).
-        E. Prada, D., Di Domenico, Y., Creff, J., Bernard, V., Sauvant-Moynot, and F.
-        Huet. "A simplified electrochemical and thermal aging model of LiFePO4-graphite
-        Li-ion batteries: power and capacity fade simulations". Journal of The
-        Electrochemical Society, 160(4), A616 (2013).
-        M. J. Lain, J., Brandon, and E. Kendrick. "Design Strategies for High Power vs.
-        High Energy Lithium Ion Cells". Batteries, 5(4), 64 (2019).
-
-Lead-acid parameter sets
---------------------------
-    * Sulzer2019 :
-        V. Sulzer, S. J. Chapman, C. P. Please, D. A. Howey, and C. W. Monroe, "Faster
-        lead-acid battery simulations from porous-electrode theory: Part I. Physical
-        model." Journal of the Electrochemical Society, 166(12), 2363 (2019).
-
+        Chang-Hui Chen, Ferran Brosa Planella, Kieran O'Regan, Dominika Gastol, W.
+        Dhammika Widanage, and Emma Kendrick. Development of Experimental Techniques for
+        Parameterization of Multi-scale Lithium-ion Battery Models. Journal of The
+        Electrochemical Society, 167(8):080534, 2020.
+        doi:10.1149/1945-7111/ab9050.Michael J. Lain, James Brandon, and Emma Kendrick.
```

Comment:
@priyanshuone6 formatting needs to be fixed for when there are multiple citations